app/models/iiif_manifest_harvester.rb

```diff
@@ -0,0 +1,22 @@
+require 'iiif/presentation'
```

Comment:
Not sure if we need to worry about `frozen_string_literal` here or not, but I noticed it was added in other new files in this PR.

Reply:
👍